### ISSUE TITLE
empty efu_uri?

### ISSUE THREAD

**Issue by theyaa** [flaxsearch/BioSolr#8]
If I use a local OWL file, and leave the value for the efu_uri empty. Will the plugin be able to parse the text in the document I send to properly classify the document according to the OWL ontology?

Best.

Theyaa

**Comment by mattflax** [comment]
No, that is not something this plugin can do. I suggest you look at the Zooma tool, which may be useful to you. See http://www.ebi.ac.uk/spot/zooma/ for more details.

Thanks.

**Comment by dsmiley** [comment]
Sorry this may be out of scope of this repo/issues here but related to Zooma... is the code behind that available?  I was poking around the site but I didn't notice it.

I developed something called the [SolrTextTagger](https://github.com/OpenSextant/SolrTextTagger) which seems to be a little similar and I'm curious about what Zooma is doing internally -- how it works.

**Comment by tburdett** [comment]
Hi David, yes - the Zooma code is here: https://github.com/EBISPOT/zooma.  I'd like to have a look at the SolrTextTagger in a bit more detail but my immediate reaction is that they're solving different problems in a manner that is potentially nicely complementary

**Comment by dsmiley** [comment]
Thanks.  Wow there's a lot of code there; it will take some time to explore to find the parts similar to the SolrTextTagger  -- which is scanning text to look for words in a dictionary, in a nutshell.  I agree that it's likely complementary.
